README.md

```diff
@@ -11,7 +11,8 @@ This package is not on the official flathub, so you'll have to download, build,
 Clone the git repo, and then run:
 
 ```
-flatpak-builder --user --install --force-clean builddir io.keybase.Keybase.yml
+flatpak remote-add --if-not-exists --user flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+flatpak-builder --user --install --force-clean builddir --install-deps-from=flathub io.keybase.Keybase.yml
```

Comment:
But why is the `--install-deps-from` needed now? It should just pick that up from all registered remotes, no? That's what it always did for me, anyway.

Reply:
Hmm, that's a good question. This was a local change for me from last September so I honestly don't entirely remember. The platform version update is from this week though. I'll remove this change.